### PR TITLE
ACI 6:Updating the additional attributes in get of Fabric, Switch and Port

### PR DESCRIPTION
**Pull request by jeevan-kamkar**
As part of this additional attributes in  added in getting of Fabric, Switch and Port  for issue #6

### CODE REVIEW

**Review by skhan-anjum — APPROVED** [review]

File: caputilities/aciClient.go [hunk 1]
```diff
@@ -83,7 +312,7 @@ func GetPortData(podID, switchID string) (*capmodel.PortResponse, error) {
 		return nil, fmt.Errorf(errMsg)
 	}
 
-	var portResponseData capmodel.PortResponse
+	var portResponseData capmodel.Helath
```
Comment (a-ajith) [inline caputilities/aciClient.go:315]: Shouldn't it be Health?

**Review by a-ajith — CHANGES_REQUESTED** [review]

File: main.go [hunk 2]
```diff
@@ -242,15 +244,64 @@ func intializeACIData() {
...
+	}
+	switchData.FirmwareVersion = switchRespData.SystemAttributes.Version
+	switchChassisData, err := caputilities.GetSwitchChassisInfo(fabricNodeData.PodId, fabricNodeData.NodeId)
+	if err != nil {
+		log.Fatal("Unable to get the Switch Chassis info for node " + fabricNodeData.NodeId + " :" + err.Error())
+	}
+	switchData.Manufacturer = switchChassisData.IMData[0].SwitchChassisData.Attributes["vendor"].(string)
+	switchData.Model = switchChassisData.IMData[0].SwitchChassisData.Attributes["model"].(string)
+	log.Info("Switch Data")
+	log.Info(switchData)
```
Comment (irappa-pattar) [inline main.go:305]: remove one of them

**Review by irappa-pattar — CHANGES_REQUESTED** [review]

File: caputilities/aciClient.go (hunk 1 above)
Comment (jeevan-kamkar) [reply caputilities/aciClient.go]: updated

**Review by a-ajith — APPROVED** [review]

File: main.go (hunk 2 above)
Comment (jeevan-kamkar) [reply main.go]: updated

**Review by irappa-pattar — APPROVED** [review]

**Review by bharath-b-hpe — APPROVED** [review]

**Review by Bharath-KKB — APPROVED** [review]

**Pull request merged**